plugins/hydra_joblib_launcher/hydra_plugins/hydra_joblib_launcher/conf/hydra/launcher/joblib.yaml

```diff
@@ -0,0 +1,18 @@
+hydra:
+  launcher:
+    class: hydra_plugins.hydra_joblib_launcher.HydraJoblibLauncher
+    params:
+      joblib: ${hydra.joblib}
+
+  joblib:
+    n_jobs: -1  # maximum number of concurrently running jobs. if -1, all CPUs are used
```

Comment:
Please put comments above field, it makes it easier to read:
```yaml
# maximum number of concurrently running jobs. if -1, all CPUs are used
n_jobs: -1

# allows to hard-code backend, otherwise inferred based on prefer and require
backend: null

# processes or threads, soft hint to choose backend
prefer: processes

```

Reply:
Done